src/components/Characters/Characters.test.jsx

```diff
@@ -0,0 +1,341 @@
+import { render, screen } from "@testing-library/react";
+import { MemoryRouter  } from "react-router-dom";
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import App from '../../App.jsx';
+
+const server = setupServer(
+    rest.get('https://rickandmortyapi.com/api/character?page=1', (req, res, ctx) => {
+        return res(
+            ctx.json({
```

Comment:
to cut down on test file size, you can always
1. save the mocked data in a data.json file and then import the file into the mock server, or
2. delete any irrelevant data from the mock response (i.e. make the mocked data fit the most basic shape of the data expected within your app